### PR TITLE
Root ulimit increase

### PR DESCRIPTION
**Pull request by sehqlr**
This fix is for #775.

### CODE REVIEW

File: roles/common/tasks/main.yml [hunk 1]
```diff
@@ -81,5 +81,19 @@
...
+- name: increase ulimit for root
+  sudo: yes
+  copy:
+    dest: /etc/security/limits.d/10-root-nofiles.conf
+    src: 10-root-nofiles.conf
+
+- name: sysctl max file limit
+  sudo: yes
+  sysctl:
+    name: fs.file-max
```
Comment (stevendborrelli) [inline roles/common/tasks/main.yml]: Do we need to run `sysctl -p` in a handler after setting this?

Comment (sehqlr) [inline roles/common/tasks/main.yml]: That's what the reload does on line#96

**Comment by siddharthist** [comment]
I'm getting this on GCE:
```
TASK: [common | sysctl max file limit] **************************************** 
failed: [lb-mi-worker-001] => {"failed": true, "parsed": false}
BECOME-SUCCESS-tljbpbchbdgegktmdajorzllmtlzoueh
Traceback (most recent call last):
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.02-25747035005906/sysctl", line 1936, in <module>
    main()
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.02-25747035005906/sysctl", line 335, in main
    result = SysctlModule(module)    
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.02-25747035005906/sysctl", line 118, in __init__
    self.process()
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.02-25747035005906/sysctl", line 157, in process
    elif not self._values_is_equal(self.proc_value, self.args['value']):
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.02-25747035005906/sysctl", line 178, in _values_is_equal
    b = b.split()
AttributeError: 'int' object has no attribute 'split'
OpenSSH_7.1p1, OpenSSL 1.0.2d 9 Jul 2015
debug1: Reading configuration data /home/lb/.ssh/config
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: auto-mux: Trying existing master
debug1: mux_client_request_session: master session id: 2
Shared connection to 104.197.192.251 closed.

failed: [lb-mi-control-01] => {"failed": true, "parsed": false}
BECOME-SUCCESS-jdtwvrddgwkkpufvfgrvhwrmddwpflxc
Traceback (most recent call last):
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.03-160072038421683/sysctl", line 1936, in <module>
    main()
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.03-160072038421683/sysctl", line 335, in main
    result = SysctlModule(module)    
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.03-160072038421683/sysctl", line 118, in __init__
    self.process()
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.03-160072038421683/sysctl", line 157, in process
    elif not self._values_is_equal(self.proc_value, self.args['value']):
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.03-160072038421683/sysctl", line 178, in _values_is_equal
    b = b.split()
AttributeError: 'int' object has no attribute 'split'
OpenSSH_7.1p1, OpenSSL 1.0.2d 9 Jul 2015
debug1: Reading configuration data /home/lb/.ssh/config
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: auto-mux: Trying existing master
debug1: mux_client_request_session: master session id: 2
Shared connection to 104.197.202.148 closed.

failed: [lb-mi-control-03] => {"failed": true, "parsed": false}
BECOME-SUCCESS-jldeaneivwwsijvgmqpxgmakpvkakboy
Traceback (most recent call last):
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.04-220816755981598/sysctl", line 1936, in <module>
    main()
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.04-220816755981598/sysctl", line 335, in main
    result = SysctlModule(module)    
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.04-220816755981598/sysctl", line 118, in __init__
    self.process()
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.04-220816755981598/sysctl", line 157, in process
    elif not self._values_is_equal(self.proc_value, self.args['value']):
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.04-220816755981598/sysctl", line 178, in _values_is_equal
    b = b.split()
AttributeError: 'int' object has no attribute 'split'
OpenSSH_7.1p1, OpenSSL 1.0.2d 9 Jul 2015
debug1: Reading configuration data /home/lb/.ssh/config
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: auto-mux: Trying existing master
debug1: mux_client_request_session: master session id: 2
Shared connection to 104.197.238.121 closed.

failed: [lb-mi-worker-002] => {"failed": true, "parsed": false}
BECOME-SUCCESS-ixlxryppvkpbapzuwgjxgyageogwalen
Traceback (most recent call last):
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.02-144798178103108/sysctl", line 1936, in <module>
    main()
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.02-144798178103108/sysctl", line 335, in main
    result = SysctlModule(module)    
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.02-144798178103108/sysctl", line 118, in __init__
    self.process()
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.02-144798178103108/sysctl", line 157, in process
    elif not self._values_is_equal(self.proc_value, self.args['value']):
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.02-144798178103108/sysctl", line 178, in _values_is_equal
    b = b.split()
AttributeError: 'int' object has no attribute 'split'
OpenSSH_7.1p1, OpenSSL 1.0.2d 9 Jul 2015
debug1: Reading configuration data /home/lb/.ssh/config
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: auto-mux: Trying existing master
debug1: mux_client_request_session: master session id: 2
Shared connection to 104.197.192.43 closed.

failed: [lb-mi-control-02] => {"failed": true, "parsed": false}
BECOME-SUCCESS-bqjtjojztlbyoewehwcboudjqibipomt
Traceback (most recent call last):
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.02-6749504552008/sysctl", line 1936, in <module>
    main()
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.02-6749504552008/sysctl", line 335, in main
    result = SysctlModule(module)    
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.02-6749504552008/sysctl", line 118, in __init__
    self.process()
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.02-6749504552008/sysctl", line 157, in process
    elif not self._values_is_equal(self.proc_value, self.args['value']):
  File "/home/centos/.ansible/tmp/ansible-tmp-1447121354.02-6749504552008/sysctl", line 178, in _values_is_equal
    b = b.split()
AttributeError: 'int' object has no attribute 'split'
OpenSSH_7.1p1, OpenSSL 1.0.2d 9 Jul 2015
debug1: Reading configuration data /home/lb/.ssh/config
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: auto-mux: Trying existing master
debug1: mux_client_request_session: master session id: 2
Shared connection to 104.197.236.0 closed.
```

**Comment by stevendborrelli** [comment]
It worked for me on openstack. What version of ansible are you using? 
```
# ansible-playbook --version
ansible-playbook 1.9.2
```

**Comment by siddharthist** [comment]
@stevendborrelli That's totally it: 
```
$ ansible-playbook --version
ansible-playbook 1.9.0.1
```

**Comment by stevendborrelli** [comment]
I know this is merged, but even after a reboot, I don't see the root ulimit increasing: 
```
ulimit -a | grep files
open files                      (-n) 1024
```

**Comment by sehqlr** [comment]
Hm.. I tested it out on my cluster in london just now, and I got this:
```
[root@shatfiel-lon1-test-control-01 cloud-user]# ulimit -a | grep files
open files                      (-n) 65000
```

**Comment by sehqlr** [comment]
I wonder if it has something to do with the reload? I set the ansible option to true..

**Comment by sehqlr** [comment]
Do I need to add a tag for this?

**Pull request merged**